notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>Note that this may not be correct, since there can be some dependence of acceptance with off-axis angle. With real data, statistics should be enough to make a large (e.g. 1 deg) value unnecessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>I just removed this, had to do with the checks I was doing to compare pyirf/lstchain. </p>